### PR TITLE
Update to @zk-email/circuits v2

### PR DESCRIPTION
**Pull request by 0xSachinK**

### CODE REVIEW

File: circuits-circom/package.json [hunk 1]
```diff
@@ -21,7 +21,10 @@
   "scripts": {
     "gen-input": "npx tsx ./scripts/generate_input.ts",
     "test": "mocha",
-    "build": "tsc"
+    "build": "tsc",
+    "compile-receive": "CIRCUIT_NAME=venmo_receive && cd scripts && ./1_compile.sh && cd ..",
```
Comment (richardliang) [inline circuits-circom/package.json:25]: nice

File: circuits-circom/venmo_receive.circom [hunk 2]
```diff
@@ -7,10 +7,7 @@ include "./regexes/venmo_payer_id.circom";
 include "./regexes/venmo_timestamp.circom";
 
 template VenmoReceiveEmail(max_header_bytes, max_body_bytes, n, k, pack_size) {
-    assert(max_header_bytes % 64 == 0);
```
Comment (richardliang) [inline circuits-circom/venmo_receive.circom:10]: Any reason to get rid of these?

File: circuits-circom/venmo_receive.circom [hunk 3]
```diff
@@ -88,13 +97,10 @@ template VenmoReceiveEmail(max_header_bytes, max_body_bytes, n, k, pack_size) {
...
-// This makes modulus, order_id and signature public. Signature being public means it allows the mailserver to trace who the offender is.
-
 // Args:
 // * max_header_bytes = 1024 is the max number of bytes in the header
 // * max_body_bytes = 6400 is the max number of bytes in the body after precomputed slice
 // * n = 121 is the number of bits in each chunk of the modulus (RSA parameter)
 // * k = 17 is the number of chunks in the modulus (RSA parameter)
 // * pack_size = 7 is the number of bytes that can fit into a 255ish bit signal (can increase later)
-component main { public [ modulus, signature, order_id ] } = VenmoReceiveEmail(1024, 6400, 121, 17, 7);
+component main { public [ order_id ] } = VenmoReceiveEmail(1024, 6400, 121, 17, 7);
```
Comment (richardliang) [inline circuits-circom/venmo_receive.circom:106]: Assuming we're now verifying the modulus_hash vs modulus array in the smart contracts

https://github.com/zkemail/zk-email-verify/blob/main/packages/circuits/email-verifier.circom#L108-L109

**Review by richardliang — COMMENTED** [review]
TBD on nullifier

File: circuits-circom/venmo_receive.circom (hunk 2 above)
Comment (0xSachinK) [reply circuits-circom/venmo_receive.circom:10]: All of this has been added to the imported template now.

File: circuits-circom/venmo_receive.circom (hunk 3 above)
Comment (0xSachinK) [reply circuits-circom/venmo_receive.circom:106]: Yup.

**Pull request merged**